### ISSUE TITLE
Missing current conditions icon and visibility after upgrade to 1.2.

### ISSUE THREAD

**Issue by ChrisM510** [poblabs/weewx-belchertown#447]
Please don't delete these questions! They are asked almost with every issue - answering them as below saves us time from asking them!

**First step**
Try to debug the skin yourself. If you can't figure it out, please include the debug information (text, screenshots, etc.). See this guide on debugging the skin. https://github.com/poblabs/weewx-belchertown#how-to-use-debug

**Describe the bug**

My current conditions icons are now missing, both on the main page, as well as the mini-conditions icon on other sub-pages.  My visibility now reads "NaN"  Both items were working before my upgrade today.  All other items are working as expected.  For the main landing page, the icon is completely missing (no broken image placeholder).  For the mini-conditions icon, I have a broken image placeholder, but the URL for the image just has the folder, not the filename (see screenshot).

**Link to your website (IMPORTANT)**

***not publicly available, local only/VPN required***

**Version of the skin you're using**

1.2 - Aries provider

**To Reproduce**
1. Open WeeWX main page

**Expected behavior**
I expected the icon to be present

**Screenshots**

https://ibb.co/fC2jnvL
https://ibb.co/tLqjZS3

**Device Information (please complete the following information):**
 - Device: Desktop and phone
 - OS: Windows and Android
 - Browser Chrome, Firefox, Edge

**Additional context**
Add any other context about the problem here. Maybe a potential fix?

**HELP WANTED**
If you have a fix for this, please submit a pull request against the development branch!

**Comment by poblabs** [comment]
Did you recently change forecast providers?

If so, delete your forecast.json file from your HTML folder. Restart weewx, and wait a couple archive intervals for it to be re-generated.

**Comment by ChrisM510** [comment]
Thanks, but I just tried that - no luck.  I stopped WeeWX, deleted /var/www/html/json/forecast.json, restarted WeeWX, then executed "sudo wee_reports" and the forecast.json file was recreated.  Tried Chrome and Firefox with incognito/private windows, same problem.

**Comment by poblabs** [comment]
Any errors in syslog? Any errors in the dev console (see how to debug the skin). Attach a copy of the forecast.json.

**Comment by ChrisM510** [comment]
Nothing is jumping out at me on the dev console, but then I am not sure I would know what is an error.  I attached a text file with the last 800 lines of my syslog, in addition to forecast.json.

[forecast.zip](https://github.com/poblabs/weewx-belchertown/files/5447687/forecast.zip)
[syslog-800lines.txt](https://github.com/poblabs/weewx-belchertown/files/5447705/syslog-800lines.txt)

BTW, thanks for helping...much appreciated!

**Comment by poblabs** [comment]
Everything checks out so far. Do you have the images folder? Within is there the cloudy.png file?

**Comment by ChrisM510** [comment]
Yes.  I even manually tried getting to it by typing the URL into the browser, and it worked.

**Comment by ChrisM510** [comment]
I have manually opened up my RPi for external traffic if you want to look: http://weewx.clmnetwork.com:8080

**Comment by poblabs** [comment]
Looks like you might have an http vs. https mismatch issue going on? I'm getting errors on your http link, that it cannot load images from https due to an invalid certificate. Maybe try specifying everything as http?

**Comment by ChrisM510** [comment]
But I am not using https...  Where should I look to verify?  I double-checked my weewx.conf file...everything is http only.

**Comment by ChrisM510** [comment]
In Firefox, if you go to the Graphs section and then find the broken icon in the upper left corner, right-click, and choose View Image Info, you only get a URL for the images folder.  If you do the same thing on your own site, you get the full URL for the png file.

**Comment by michaelundwd** [comment]
I had this problem that was solved by setting forecast_aeris_use_metar = 0 in skin.conf.  See #393 for more details.
Michael

**Comment by michaelundwd** [comment]
Sorry, got that the wrong way round.  I set forecast_aeris_use_metar =1 to fix it. 
Michael

**Comment by ChrisM510** [comment]
Thanks, but I just confirmed that my skin.conf file was already set to that.
![Capture1](https://user-images.githubusercontent.com/10404766/97358544-c5924280-1871-11eb-8dfa-2b9e84006057.JPG)

**Comment by poblabs** [comment]
Not sure. Is your pi set to do some https? There were 2 errors I had in the
console debug when I went to your site. I can send those later today

**Comment by ChrisM510** [comment]
Here are my weewx.conf and skin.conf files...maybe these will help?  I never had anything set to https before the upgrade.  All I did today was run the upgrade install, then go into weewx.conf and change over from darksky to aeris.

[Config files.zip](https://github.com/poblabs/weewx-belchertown/files/5447993/Config.files.zip)

**Comment by poblabs** [comment]
Great news!
